cwms-data-api/src/main/java/cwms/cda/data/dto/Unit.java

```diff
@@ -23,7 +23,7 @@
 @FormattableWith(contentType = Formats.JSONV2, formatter = JsonV2.class, aliases = {Formats.DEFAULT, Formats.JSON})
 @FormattableWith(contentType = Formats.XMLV2, formatter = XMLv2.class, aliases = {Formats.XML})
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
-public final class Unit extends CwmsDTO
+public final class Unit extends CwmsDTOBase
```

Comment:
I'm not certain if this class was implemented with knowledge of the database notion of DisplayUnits? See source for AV_DISPLAY_UNIT which brings in office specific parameter units

Reply:
@RyanM-RMA - I'll let you chime in here.

Reply:
It was not implemented with that knowledge.  The table and view it's using, AT_UNIT_ALIAS, and AV_UNIT, both contain office IDs, so we could make a ticket to update them to include those office ids.  Seems a bit out of scope for this work though.

Reply:
Actually, looking at the original output from this (it was using CWMS_CAT.RETRIEVE_UNITS_F before), that output didn't include the office id, which is probably why it was ignored in the changes I made.

Reply:
The importance of office ID as used in display vs the non-office ID CWMS defaults is that the unit will be the one that the district has determined to be their default. @RyanM-RMA - How is that out of scope?

Reply:
I say out of scope because it's related to recent maintenance ticket work, not our fixed cost task.

Reply:
This is the ticket for that work.
https://github.com/USACE/cwms-data-api/issues/498